### ISSUE TITLE
[BUG] Warning message on the forms

### ISSUE THREAD

**Issue by Pradumnasaraf** [EddieHubCommunity/LinkFree#7855]
### Description

As we mentioned, please delete the `username.json` file and the `username` folder. However, if someone provides a testimonial to that user, it will again create a folder with that person's username, which may cause confusion for them.

### Screenshots

<img width="1294" alt="Screenshot 2023-06-29 at 11 45 50 PM" src="https://github.com/EddieHubCommunity/LinkFree/assets/51878265/3a9aff73-0667-49c9-963d-8cd568be9a15">

### Additional information

A potential solution could be supporting testimonials through the UI.

**Comment by amandamartin-dev** [comment]
I like @Pradumnasaraf suggestion of supporting testimonials via the UI.  In the meantime, perhaps there is a way to hide this notification by checking in the path if it leads to a profile.json or to testimonials.

**Comment by eddiejaoude** [comment]
Yep moving testimonials to forms is a longer term solution but we need to think of a way to also solve this for people who add testimonials via the issue which will create a json file 🤔

**Comment by amandamartin-dev** [comment]
> Yep moving testimonials to forms is a longer term solution but we need to think of a way to also solve this for people who add testimonials via the issue which will create a json file 🤔

Would a simple conditional statement work to hide/show the notification?

Like if profile.json exists, show notification, else hide

**Comment by eddiejaoude** [comment]
At the moment I think we check for the `[username].json` file. But the testimonial is in `[username]/testimonial/[testimonial-username].json` file - so if they remove the folder, and someone adds a testimonial afterwards via the issue template, the folder and file will reappear and the receiver will have to keep deleting this folder 🤔
